docs/eventing/samples/kafka/source/README.md

```diff
@@ -0,0 +1,202 @@
+   2019/03/19 22:25:54 Registering Components.
+   2019/03/19 22:25:54 Setting up Controller.
+   2019/03/19 22:25:54 Adding the Apache Kafka Source controller.
+   2019/03/19 22:25:54 Starting Apache Kafka controller.
+   ```
+
+### Apache Kafka Topic (Optional)
+
+1. If using Strimzi, you can can set a topic modifying
+   `source/kafka-topic.yaml` with your desired:
+
+- Topic
+- Cluster Name
+- Partitions
+- Replicas
+
+  ```yaml
+  apiVersion: kafka.strimzi.io/v1alpha1
+  kind: KafkaTopic
+  metadata:
+    name: knative-demo-topic
+    namespace: kafka
+    labels:
+      strimzi.io/cluster: my-cluster
+  spec:
+    partitions: 3
+    replicas: 1
+    config:
+      retention.ms: 7200000
+      segment.bytes: 1073741824
+  ```
+
+2. Deploy the `KafkaTopic`
+
+   ```shell
+   $ kubectl apply -f kafka/source/samples/strimzi-topic.yaml
+   kafkatopic.kafka.strimzi.io/knative-demo-topic created
+   ```
+
+3. Ensure the `KafkaTopic` is running.
+
+   ```shell
+   $ kubectl -n kafka get kafkatopics.kafka.strimzi.io
+   NAME                 AGE
+   knative-demo-topic   16s
+   ```
+
+### Create the Event Display service
+
+1. Build and deploy the Event Display Service.
+   ```
+   $ kubectl apply --filename source/samples/event-display.yaml
+   ...
+   service.serving.knative.dev/event-display created
+   ```
+2. Ensure that the Service pod is running. The pod name will be prefixed with
+   `event-display`.
+   ```
+   $ kubectl get pods
+   NAME                                            READY     STATUS    RESTARTS   AGE
+   event-display-00001-deployment-5d5df6c7-gv2j4   2/2       Running   0          72s
+   ...
+   ```
+
+#### Apache Kafka Event Source
+
+1. Modify `source/event-source.yaml` accordingly with bootstrap
+   servers, topics, etc...
+
+   NOTE: If using an internal Apache Kafka cluster, you may need to ensure
+   you've specified the correct variables set in any `KafkaTopic` and
+   `event-source`. For example, the following source could be used:
+
+   ```yaml
+   apiVersion: sources.eventing.knative.dev/v1alpha1
+   kind: KafkaSource
+   metadata:
+     name: kafka-source
+   spec:
+     consumerGroup: knative-group
+     bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092 #note the kafka namespace
+     topics: knative-demo-topic
+     sink:
+       apiVersion: serving.knative.dev/v1alpha1
+       kind: Service
+       name: event-display
+   ```
+
+2. Build and deploy the event source.
+   ```
+   $ kubectl apply -f kafka/source/samples/event-source.yaml
+   ...
+   kafkasource.sources.eventing.knative.dev/kafka-source created
+   ```
+3. Check that the event source pod is running. The pod name will be prefixed
+   with `kafka-source`.
+   ```
+   $ kubectl get pods
+   NAME                                  READY     STATUS    RESTARTS   AGE
+   kafka-source-xlnhq-5544766765-dnl5s   1/1       Running   0          40m
+   ```
+4. Ensure the Apache Kafka Event Source started with the necessary
```

Comment:
just 1.
Is there a label that we could use for logs? As in kubectl logs -selector...?